tests/models/aria/test_modeling_aria.py

```diff
@@ -0,0 +1,700 @@
+# coding=utf-8
```

Comment:
maybe also `test_processing_aria` and `test_image_processing`, as it is not same as most vision processors and has special treatment for images